azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -158,5 +170,4 @@ private String getJobOrFlowName(final SlaOption slaOption) {
       return flowName;
     }
   }
-
-}
+}
```

Comment:
Why?

Reply:
why for which part?

Reply:
I mean why removing the newline at the end of the file? Github even adds a warning about it.